### PR TITLE
quaybackstop: add backstop GC command

### PR DESCRIPTION
**Pull request by hdonnay**
`Quaybackstop` is a helper command to perform "backstop" garbage collection for a Clair+Quay system.

### CODE REVIEW

**Comment by hdonnay** [comment]
No tests other than some dry-runs on my laptop. If interested, a command like the following should run the "manifests" and "select" steps of the pipeline:
```
quaybackstop \
        -clair-gabi 'https://gabi-clair-indexer-prod.apps.rosa.clairp02ue1.ts2n.p3.openshiftapps.com/'\
        -clair-gabi-auth=$(oc --kubeconfig ~wc/kubeconfig/clair-prod whoami -t)\
        -quay-gabi 'https://gabi-quayio-prod.apps.quayp04ue2.k9x1.p1.openshiftapps.com/'\
        -quay-gabi-auth=$(oc --kubeconfig ~wc/kubeconfig/quay-prod whoami -t)\
        -n
```

**Comment by hdonnay** [comment]
I think this is good to merge, and will need to be merged before it can get used on quayio staging

File: contrib/openshift/manifests/backstop.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    app: clair
+labels:
+  app: clair
+objects:
+- apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: quaybackstop
+  spec:
+    schedule: '23 1 * * 1'
```
Comment (crozzy) [inline contrib/openshift/manifests/backstop.yaml:16]: :thinking:

**Review by crozzy — APPROVED** [review]
SGTM

File: contrib/openshift/manifests/backstop.yaml (hunk 1 above)
Comment (hdonnay) [reply contrib/openshift/manifests/backstop.yaml:16]: Think there's a better time to run than `01:23Z`?

**Pull request merged**